src/lib/viewers/doc/_docBase.scss

```diff
@@ -295,6 +295,17 @@ $thumbnail-sidebar-width: 191px; // Extra pixel to account for sidebar border
         }
     }
 
+    //overwriting background image in pdf_viewer.css. Was causing 404's
+    /* stylelint-disable declaration-no-important */
+    .pdfViewer .page.loadingIcon {
```

Comment:
Should there be a space between `.page` and `.loadingIcon`? It's a child element, right?

Reply:
Believe it or not no. That's how pdf.js v3.6.7.2 has it. I can take out line 300 but I think we need the other one. Check this out https://github.com/mozilla/pdf.js/blob/v3.6.172/web/pdf_viewer.css#L162